qa/tasks/barbican.py

```diff
@@ -357,7 +358,7 @@ def create_secrets(ctx, config):
             if not (barbican_sec_resp.status >= 200 and
                     barbican_sec_resp.status < 300):
                 raise Exception("Cannot create secret")
-            barbican_data = json.loads(barbican_sec_resp.read())
+            barbican_data = json.loads(six.ensure_str(barbican_sec_resp.read()))
```

Comment:
Interesting that recent version of json.loads in py2 is always returning data with strings of `unicode` type what ever is provided as a json.